src/core/main.c

```diff
@@ -1364,7 +1363,6 @@ int main(int argc, char *argv[]) {
 
                 /* Running outside of a container as PID 1 */
                 arg_system = true;
-                make_null_stdio();
```

Comment:
Because otherwise colors_enabled() doesn't work since it looks at the device connected to stdio.